client/ayon_flame/api/plugin.py

```diff
@@ -621,19 +630,45 @@ def _convert_to_marker_data(self):
                 `tag_hierarchy_data` will be set only once for every
                 clip which is not hero clip.
                 """
-                _hero_data = deepcopy(hero_data)
-                _hero_data.update({"heroTrack": False})
+                _distrib_data = deepcopy(hero_data)
+                _distrib_data.update({"heroTrack": False})
                 if _in <= self.clip_in and _out >= self.clip_out:
```

Comment:
You don't need to copy or start any logic before the condition as the variable is not used anywhere and the copy is waste of resources.
```suggestion
                if _in <= self.clip_in and _out >= self.clip_out:
                    _distrib_data = deepcopy(hero_data)
                    _distrib_data.update({"heroTrack": False})
```

BTW much better would be to reverse the condition with continue.
```
if _in > self.clip_in or _out < self.clip_out:
    continue
```